community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/MessageProcessingHandler.java

```diff
@@ -139,8 +140,23 @@ private void publishError( BoltResponseMessageHandler<IOException> out, Neo4jErr
         }
         catch ( PackOutputClosedException e )
         {
-            // we tried to write error back to the client and realized that the underlying channel is closed
-            // log a warning, client driver might have just been stopped and closed all socket connections
+            // Can't write error to the client, because the connection is closed.
+            // Very likely our error is related to the connection being closed.
+
+            // If the error is that the transaction was terminated, then the error is a side-effect of
+            // us cleaning up stuff that was running when the client disconnected. Log a warning without
+            // stack trace to highlight clients are disconnecting while stuff is running:
+            if ( error.status() == Status.Transaction.Terminated )
```

Comment:
Worth also checking `LockClientStopped` because it has pretty much the same meaning.
I'd also vote for having a static set of all "OK" statuses and doing `set.contains(error.status())` here.